sources/packages/backend/libs/sims-db/src/entities/ecert-feedback-error.model.ts

```diff
@@ -0,0 +1,45 @@
+import { Column, Entity, PrimaryGeneratedColumn } from "typeorm";
+import { OfferingIntensity } from ".";
+import { TableNames } from "../constant";
+
+/**
+ * Errors that can be received in the e-Cert feedback file.
+ */
+@Entity({ name: TableNames.ECertFeedbackErrors })
+export class ECertFeedbackError {
+  @PrimaryGeneratedColumn()
+  id: number;
```

Comment:
Minor, `id` is missing the comment.

Reply:
Was this comment accepted? I can't see the comment.
It will not be a blocker, just checking.

Reply:
I got missed. I will add the comment.